rules/sinks/storages/cookiemanager/javascript.yaml

```diff
@@ -2,7 +2,8 @@ sinks:
   - id: Storages.Web.Cookie
     name: Web Storage Cookie
     patterns:
-      - "(?i).*(getCookie|setCookie|deleteCookie)"
+      - "(?i).*(getCookie|setCookie|deleteCookie|removeCookie|useCookies)"
+      - "(?i)(cookie).*"
```

Comment:
Can this rule coincide nodes with the sourceRule for Cookie. We may say unexpected result in that scenario

Reply:
I think for frontend we can try it where we will get specific PIIs which will be passed as Cookies. If its causing issue, will remove this rule.